src/lib/validations/is-multi-site-sql-dump.js

```diff
@@ -28,23 +28,21 @@ export function sqlDumpLineIsMultiSite( line: string ): boolean {
 	return false;
 }
 
-export function isMultiSiteDumpFile( fileName: string ): Promise<boolean> {
-	return new Promise( async resolve => {
-		const readInterface = await getReadInterface( fileName );
-		readInterface.on( 'line', line => {
-			const result = sqlDumpLineIsMultiSite( line );
-			if ( true === result ) {
-				resolve( true );
-			}
-		} );
+export async function isMultiSiteDumpFile( fileName: string ): Promise<boolean> {
+	const readInterface = await getReadInterface( fileName );
+	readInterface.on( 'line', line => {
+		const result = sqlDumpLineIsMultiSite( line );
+		if ( true === result ) {
+			return true;
+		}
+	} );
 
-		readInterface.on( 'error', () => {
-			exit.withError(
-				'An error was encountered while reading your SQL dump file.  Please verify the file contents.'
-			);
-		} );
-		// Block until the processing completes
-		await new Promise( resolveBlock => readInterface.on( 'close', resolveBlock ) );
-		resolve( false );
+	readInterface.on( 'error', () => {
+		exit.withError(
+			'An error was encountered while reading your SQL dump file.  Please verify the file contents.'
+		);
 	} );
+	// Block until the processing completes
+	await new Promise( resolveBlock => readInterface.on( 'close', resolveBlock ) );
+	return false;
```

Comment:
This refactor is broken. How does it return anything other than false?

Reply:
There are no tests covering it either

Reply:
Hmmm ... is this function used at all??

Reply:
Yes, you are right - I have completely missed that the promise was resolved from a callback.

Removed the function, as there are no references to it. It looks like the code that used to use it was refactored to use `sqlDumpLineIsMultiSite()` instead.